README.md

```diff
@@ -73,79 +49,29 @@ I can see whizzy animations in the app
-```
-
-Note how the `apiRequestUrl` parameter value is just the request you would have made to the Guardian API, minus `api-key`.
-
-### Aylien text summarisation API example
-
-**Please stub your tests to avoid exceeding the API rate limit**
-
-If you wanted to use the Aylien API to summarise an article by Bret Victor, this is the cURL request you might make.  Notice how it has headers to authenticate with the Aylien API.
-
-```
-curl "https://api.aylien.com/api/v1/summarize?url=http://worrydream.com/MediaForThinkingTheUnthinkable/note.html" \
-  -H "X-AYLIEN-TextAPI-Application-ID: APPLICATION_ID" \
-  -H "X-AYLIEN-TextAPI-Application-Key: SECRET_APPLICATION_KEY"
-```
-
-To make this request via the Makers News Summary API with cURL, you could do something like this.
-
-```
-curl "http://news-summary-api.herokuapp.com/aylien?apiRequestUrl=https://api.aylien.com/api/v1/summarize?url=http://worrydream.com/MediaForThinkingTheUnthinkable/note.html"
-```
+* Index.js is the controller: 
+  * It creates new instances of NewsApi, NewsModel and newsView classes. 
+  * It calls api.retrieveNewsData from the NewsApi class
 
-Note how the `apiRequestUrl` parameter is just the request you would have made to the Aylien API.  Notice how you don't have to send authentication headers.
+* NewsApi class:
+  * retrieveNewsData accepts a callback function and fetches the data from the Makers news api. 
+  * The call back function is defined in index.js as a function that takes in the newsData retrieved from the fetch and calls view.displayNews(newsData)
 
-### Code
+* News View Class:
+  * Is constructed with the model and api instance that is instantiated in index.js.
+  * Has a displayNews method that takes in the fetched newsData, and passes it into this.model.getNews() which turns it into an array of news. 
+  * displayNews() iterates throuogh each item in the newsArray and creates a div element showing the title and thumbnail of the article. 
 
-If you're interested, you can see the code for the News Summary API in this repo: https://github.com/makersacademy/news-summary-api
+* News Model Class:
+  * Has a getNews(newsData) method that takes in the fetched newsData object and grabs only the newsData.response.results array. It returns the array, which is then used in my newsView displayNews method. 
 
-## Resources
+### Next Steps
 
-* [Guardian newspaper API homepage](http://open-platform.theguardian.com/documentation/)
-* [Aylien text summary API docs](http://docs.aylien.com/docs/summarize)
-* cURL [man page](https://curl.haxx.se/docs/manpage.html)
-* [Hurl](https://www.hurl.it/), a web interface for sending HTTP requests
+I have not implemented CSS as I had some issues with the Makers News Api. I tried signing up to the Guardian and getting my own API key and storing this in an .env file in gitignore.  After trying this I had issues with bundling dotenv with esbuild. 
```

Comment:
Yeah the dotenv thing was a whole nightmare. I think Alfonso managed to find a way to do it though using webpack-dotenv, you could ask him if you want to know